agent/README.md

```diff
@@ -80,7 +80,7 @@ Tips：
 ```
 docker run -d --name="citamon_agent_cita_exporter_1337" \
 --pid="host" \
--p 1923:1920 \
+-p 1923:1923 \
```

Comment:
@sunfjun please change `-p 1923:1920` \ to `-p 1920:1923 \`, and undo the line 95 change

Reply:
and squash your commits to 1 commit

Reply:
Has been updated.